nebula_ros/launch/continental_launch_all_hw.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0"?>
+<launch>
+    <arg name="launch_hw" default="true"/>
+    <arg name="sensor_model" default="ARS548"/>
+    <arg name="frame_id"  default="continental"/>
+    <arg name="base_frame"  default="base_link"/>
+    <arg name="object_frame"  default="base_link"/>
+
+    <arg name="odometry_topic" default="odometry_input"/>
+    <arg name="acceleration_topic"  default="acceleration_input"/>
+    <arg name="steering_angle_topic"  default="steering_angle_input"/>
+
+    <!-- Ethernet based sensor: ARS548 -->
+    <arg name="sensor_ip" default="10.13.1.114" description="Radar Sensor IP"/>
+    <arg name="host_ip" default="10.13.1.166" description="Broadcast IP from Sensor"/>
+    <arg name="multicast_ip" default="224.0.2.2" description="Multicast Sensor IP"/>
+    <arg name="data_port" default="42102" description="Radar Data Port"/>
+    <arg name="configuration_host_port" default="42401" description="Radar host configuration port"/>
+    <arg name="configuration_sensor_port" default="42101" description="Radar sensor configuration port"/>
+    <arg name="use_sensor_time" default="false" description="Whether to use or not the timestamp from the sensor"/>
+
+    <arg name="configuration_vehicle_length" default="4.89" description="New vehicle length"/>
+    <arg name="configuration_vehicle_width" default="1.896" description="New vehicle width"/>
+    <arg name="configuration_vehicle_height" default="2.5" description="New vehicle height"/>
+    <arg name="configuration_vehicle_wheelbase" default="2.79" description="New vehicle wheel base"/>
+
+    <!-- CAN based sensor: SRR520 -->
+    <arg name="interface" default="can4" description="Can interface"/>
+    <arg name="receiver_timeout_sec" default="0.03" description="Receiver timeout in seconds"/>
+    <arg name="sender_timeout_sec" default="0.01" description="Sender timeout int seconds"/>
+    <arg name="filters" default="0:0" description="CAN filters"/>
+    <arg name="use_bus_time" default="false" description="Whether to use the bus time for received messages or the system time"/>
+
+    <arg name="new_sensor_id" default="0" description="The new sensor id"/>
+    <arg name="new_plug_bottom" default="true" description="Whether the plug is in the bottom"/>
+    <arg name="new_cover_damping" default="0.0" description="Cover damping in dB"/>
+    <arg name="reset_sensor_configuration" default="false" description="Whether to reset all values to default when configuring the sensor"/>
```

Comment:
Please move this to #124 if possible :pray:

Reply:
Addressed in [1007571](https://github.com/tier4/nebula/pull/123/commits/100757164b9eee0ced387ed2c5276687796abbfd)